src/config-hardware/mouse-touchpad.md

```diff
@@ -0,0 +1,65 @@
+# Mouse & Touchpad
+
+Configure Mouse and Touchpad settings in `Settings` ➞ `Mouse & Touchpad`.
+
+![Mouse & Touchpad Settings](/images/config-mouse-touchpad/mouse-touchpad-settings.png)
+
+Mouse settings include Mouse Speed, Mouse Acceleration, and Natural Scrolling.
+
+Touchpad settings include enabling or disabling the touchpad entirely, Natural Scrolling, Touchpad Speed, Tap to Click, Two-finger scrolling, Edge Scrolling, and Disable while typing. Users can also set the Right or Left mouse button as the primary button.
+
+---
+
+## Testing Mouse Settings
+
+Click `Test Your Settings` in the upper right corner of the window. Use this screen to perform and observe normal navigational actions.
+
+![Mouse Testing Screen](/images/config-mouse-touchpad/test-mouse-settings.png)
+
+Scroll up and down on the mouse wheel or drive two fingers across the track pad to test scroll speed.
+
+<video autoplay loop>
+    <source src="/images/config-mouse-touchpad/test-scrolling.webm" />
+</video>
+
+Left, right, or middle-click the circle to test each assigned button and click registration speed. Test double-click speed by clicking the mouse button twice within the circle. The first click fills the outer ring, and the second click fills the circle.
+
+<video autoplay loop>
+    <source src="/images/config-mouse-touchpad/test-clicking.webm" />
+</video>
+
+## General
+
+The Primary Button performs primary actions in a graphical user interface (GUI). For right-handed users, this is usually assigned to the left mouse button.
+
+## Mouse
+
+| Option | Function |
+|--------|----------|
+| Mouse Speed | Adjust the speed of mouse cursor movement across the display when using a mouse. |
+| Mouse Acceleration | Enable or disable mouse speed increasing with sustained motion in any given direction. This allows less physical movement to reach further corners of the display. |
+| Natural Scrolling | Invert the direction content scrolls in response to the mouse wheel. When enabled, scrolling up on the mouse wheel pushes content toward the top of the window, moving the user's view to content lower on the page.
```

Comment:
If you keep the mouse and keyboard Natural Scrolling description separate, I would still suggest making a comparison to a touch device in the mouse section. (E.g. Apple's Magic Mouse and Microsoft's Arc Touch Mouse use flat touch surfaces instead of mechanical wheels; you don't necessarily need to acknowledge that possibility, but that's part of why I think it can be useful to explain it this way even to someone using a mechanical wheel.)